tools/github-membership/go.mod

```
@@ -0,0 +1,8 @@
+module github.com/trodge/magic-modules/membership-tools/tools/github-membership
+
+go 1.25
```

```suggestion
go 1.23
```
consistency with other packages